documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -776,16 +776,22 @@ spec:
     storage: <6>
       name: <storage_map> <7>
       namespace: <namespace>
-  preserveStaticIPs: <8>
+  networkNameTemplate: <"net-{{.NetworkIndex}}"> <8>
+  preserveStaticIPs: <9>
+  pvcNameTemplate: <"{{.VmName}}-disk-{{.DiskIndex}}"> <10>
```

Comment:
can you group together the 3 template parameters together ?

instead of:
```
networkNameTemplate
preserveStaticIPs
pvcNameTemplate
vms
volumeNameTemplate
```
do:

```
preserveStaticIPs
networkNameTemplate
pvcNameTemplate
volumeNameTemplate
vms
```

Reply:
@yaacov No problem, I saw that parameters were listed alphabetically in the screen captures.